packages/vector_graphics_compiler/lib/src/svg/parser.dart

```diff
@@ -1217,6 +1217,19 @@ class SvgParser {
     }
   }
 
+  double? _parseWidth(String? rawWidth) {
+    try {
+      return parseDoubleWithUnits(rawWidth);
+    } on FormatException catch (e) {
+      switch (e.message) {
+        case 'Invalid double':
+          return 1.0;
```

Comment:
I was looking for an answer to this question I had earlier:

> Is there a standard for how SVG parsing and failure handling are supposed to work, or is it client-specific?

and it appears that [the spec says to ignore invalid values](https://svgwg.org/svg2-draft/types.html#TermInvalidValue). The definition of ignored that it links to says "[the UA] treats it as if it wasn’t there at all."

Given that, I'm not clear on why there's a special wrapper returning `1.0`, rather than just using `tryParse: true` when parsing the width.

Reply:
I'm sorry, previously I tried rendering an SVG in the browser and found that if it was given an invalid value then the result rendered was the same as `stroke-width` with initial value `1` as in the following [docs](https://svgwg.org/svg2-draft/painting.html#StrokeWidth), apparently I missed testing if the value/property was not defined, 
it turns out displayed the same result, which means that if the value provided is invalid it will be ignored as stated in the [docs](https://svgwg.org/svg2-draft/types.html#TermInvalidValue).

so yes this special wrapper is not needed, adding `tryParse: true` when parsing stroke-width is enough